web_select_all_companies/i18n/nl_BE.po

```diff
@@ -0,0 +1,32 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* web_select_all_companies
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 17.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-04-15 14:44+0000\n"
+"PO-Revision-Date: 2024-04-15 14:44+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: web_select_all_companies
+#. odoo-javascript
+#: code:addons/web_select_all_companies/static/src/xml/switch_all_company_menu.xml:0
+#, python-format
+msgid "All Companies"
+msgstr "Alle bedrijven"
+
+#. module: web_select_all_companies
+#. odoo-javascript
+#: code:addons/web_select_all_companies/static/src/xml/switch_all_company_menu.xml:0
+#: code:addons/web_select_all_companies/static/src/xml/switch_all_company_menu.xml:0
+#: code:addons/web_select_all_companies/static/src/xml/switch_all_company_menu.xml:0
+#, python-format
+msgid "Select All Companies"
+msgstr "Selecteer Alle Bedrijven"
```

Comment:
You can add this to a nl.po file instead, because then, you will have translations for both nl and nl_BE. nl_BE will take nl as default.
(Also add empty line at the end)